apps/explorer/test/explorer/chain/csv_export/address_internal_transaction_csv_exporter_test.exs

```diff
@@ -91,24 +91,24 @@ defmodule Explorer.Chain.AddressInternalTransactionCsvExporterTest do
           }
         end)
 
-      assert result.transaction_hash == to_string(internal_transaction.transaction_hash)
-      assert result.index == to_string(internal_transaction.index)
-      assert result.block_number == to_string(internal_transaction.block_number)
-      assert result.block_index == to_string(internal_transaction.block_index)
-      assert result.block_hash == to_string(internal_transaction.block_hash)
-      assert result.transaction_index == to_string(internal_transaction.transaction_index)
+      assert result.transaction_hash == [[], to_string(internal_transaction.transaction_hash)]
```

Comment:
why is everything now prepended with empty lists?

Reply:
Yeah, it's introduced [here](https://github.com/blockscout/blockscout/commit/f44e4e2cc44bc99d4d129b72267d6fbee58c53af), I think it's a change in the nimble_csv lib.